app/views/entities/_entity.json.jbuilder

```diff
@@ -1,4 +1,2 @@
-# frozen_string_literal: true
-
 json.extract! entity, :id, :name, :amount, :AuthorID, :created_at, :updated_at
```

Comment:
- [ ] In reading from the  requirements: **Home page (catagories page)**
- For each category, the user can see their name, icon and the total amount of all the transactions that belongs to that category. **I notice that the total amount of  transactions of the categories is missing from your App**.
![Capture](https://github.com/fitz95/Budget_App/assets/63972670/125a6293-eb7e-4a43-850b-d3ee42b285d9)

- [ ] In reading from the  requirements: **Transactions page**
- At the top of the page the user could see the total amount for the category (sum of all of the amounts of the transactions in that category). **For this one, you got it** 👍 
![Capture1](https://github.com/fitz95/Budget_App/assets/63972670/c529fe4f-a160-469a-902f-a94f573ef432)

- [ ] **Design**: **I would advise you to follow the design as stated in the requirements**:
* Colors
* Typography: font face, size and weight.
* Layout: composition and space between elements

- **See the original design:**


![Capture](https://github.com/fitz95/Budget_App/assets/63972670/cd71703f-68f1-4dc5-a08b-7848173f06b4)

Reply:
- [ ] As stated by the previous reviwer, kindly consider to follow design guidlines (Colors, Typography: font face, size and weight, Layout: composition and space between elements) 

| Your design  | Design guidline |
| ------------- | ------------- |
| ![image](https://github.com/fitz95/Budget_App/assets/53402600/22a456a7-333c-49a3-a873-a0bd42810bef)  | ![image](https://github.com/fitz95/Budget_App/assets/53402600/b2264d42-2c57-4f9d-80ca-95b02bab1e90) |